action-groups.tf

```diff
@@ -43,3 +43,26 @@ module "cmc-pdf-fail-action-group" {
 output "pdf_action_group_name" {
   value = "${module.cmc-pdf-fail-action-group.action_group_name}"
 }
+
+// CMC Admission Failures
+
+data "azurerm_key_vault_secret" "ff4j_email_secret" {
+  name = "ff4j-admissions-failure-email"
+  vault_uri = "${data.azurerm_key_vault.cmc_key_vault.vault_uri}"
+}
+
+module "cmc-ff4j-admissions-fail-action-group" {
+  source = "git@github.com:hmcts/cnp-module-action-group"
+  location = "global"
+  env = "${var.env}"
+
+  resourcegroup_name = "${azurerm_resource_group.rg.name}"
```

Comment:
minor: `resource_group_name`

Reply:
this is used on all the definitions and these definitions are used in the action-groups.tf. Do you want me to change all of these instances? If I do, will it cause any issues with existing alerts?

Reply:
sorry Tharack - this is a module variable anyway so we can't change !!! https://github.com/hmcts/cnp-module-action-group